lib/RedisClient.js

```diff
@@ -7,11 +7,13 @@ module.exports = class RedisClient {
     * @constructor
     * @param {string} host - Redis host
     * @param {number} port - Redis port
+    * @param {string} password - Redis password
     */
-    constructor(host, port) {
+    constructor(host, port, password) {
         this._client = new Redis({
             host,
             port,
+            password,
```

Comment:
Does `password` here need to be wrapped in an object?
https://github.com/luin/ioredis/blob/master/API.md#new_Redis_new